### ISSUE TITLE
Add outputs for additional formats

### ISSUE THREAD

**Issue by bollwyvl** [conda-forge/jsonschema-feedstock#27]
It would be nice to have corresponding packages for the extras of regular expressions

```ini
[options.extras_require]
format =
    idna
    jsonpointer>1.13
    rfc3987
    strict-rfc3339
    webcolors
format_nongpl =
    idna
    jsonpointer>1.13
    webcolors
    rfc3986-validator>0.1.0
    rfc3339-validator
```

**Comment by bollwyvl** [comment]
This is now unblocked... will proceed with #39 later today...

**Comment by bollwyvl** [comment]
Thanks for the assist!
